packages/compiler/test/checker/augment-decorators.test.ts

```diff
@@ -170,6 +170,38 @@ describe("compiler: checker: augment decorators", () => {
     it("interface", () => expectTarget(`@test("target") interface Foo { }`, "Foo"));
     it("operation in interface", () =>
       expectTarget(`interface Foo { @test("target") list(): void }`, "Foo.list"));
+    it("uninstantiated template", () => expectTarget(`@test("target") model Foo<T> { }`, "Foo"));
```

Comment:
@markcowl is this true? Should this be allowed or should this also throw an error? #1354 wasn't entire clear on this.

Reply:
you should be able to augment uninstantiated templates(the same way you can decorate them in the first place)

Reply:
This test now passes.